ts/Data/Connectors/JSONConnector.ts

```diff
@@ -142,7 +142,9 @@ class JSONConnector extends DataConnector {
                     // If already loaded, clear the current rows
                     table.deleteColumns();
                     converter.parse({ data });
+
                     table.setColumns(converter.getTable().getColumns());
+                    table.setRowKeysColumn(data.length);
```

Comment:
Is it only true for JSON Connector?

Reply:
So far I have only added it for JSON-connector. I will separate tickets for the other connectors as soon as this ticket is approved.